skbio/sequence/_sequence.py

```diff
@@ -1141,6 +1164,92 @@ def __deepcopy__(self, memo):
         """
         return self._copy(True, memo)
 
+    def add(self, x, *intervals):
```

Comment:
rename it to `add_feature`? so that the name of `seq.add_feature()` is more straightforward.